### PR TITLE
Added doc on coverage and soft timeout

### PR DESCRIPTION
**Pull request by jobergum**
Please review @baldersheim & @bratseth. Not sure where to add links in to this doc @kkraune ?

### CODE REVIEW

File: documentation/graceful-degradation.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+could imagine scenarios where 10 out of 11 nodes involved in the query execution were healthy and triggered soft timeout and delivered a result while the last node was in a bad state (e.g hw issues) and could not produce
+a result at all and that would cause both <em>timeout</em> and <em>adaptive-timeout</em> to be true.</p>
+
+<p>If you are working on Results in a custom <a href="searcher-development.html">custom searcher</a> you can get the coverage information programmatically:</p>
+<pre>
+    @Override
+    public Result search(Query query, Execution execution) {
+        Result result = execution.search(query);
+        Coverage  coverage = result.getCoverage(false);
+        if (coverage != null && coverage.isDegraded()) {
```
Comment (kkraune) [inline documentation/graceful-degradation.html]: &amp;amp;

**Comment by kkraune** [comment]
maybe link from PERFORMANCE AND TUNING section in left frame. also OK to link from the ref docs to this

**Comment by jobergum** [comment]
Added links to this doc from reference and performance index and fixed &&

File: documentation/graceful-degradation.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline documentation/graceful-degradation.html]: 2019

**Pull request merged**